src/ecalc/libraries/libecalc/common/libecalc/core/consumers/legacy_consumer/component.py

```diff
@@ -265,10 +265,27 @@ def evaluate(
                         new_time_vector=variables_map.time_vector,
                     )
                 )
+
```

Comment:
looks ok, but we are a bit inconsistent in the way we create the TimeSeries here, should be done in the same way unless there is a reason not to (see above)